website/docs/wallet/withdraw-validator.md

```diff
@@ -0,0 +1,404 @@
+
+:::caution
+⚠️ We recommend doing this next step *without* an Internet connection to be maximally secure. Either turn off the internet before introducing your mnemonic for signing or migrate to an air-gapped environment to continue the following steps.
+:::
+
+Here’s the command to get started with the process. This command will **not** submit your signed message to the network yet, and will only generate the data needed for the next steps.
+
+```jsx
+python ./staking_deposit/deposit.py generate-bls-to-execution-change
+```
+
+By calling the command above, you should go through an interactive process that will ask you for the following information:
+
+1. **Your mnemonic language**. You can see the different options available, where English is one of the options, among others
+2. **The network** you wish to perform this operation for. Select the `zhejiang` network for this tutorial
+3. Enter your **mnemonic** next
+4. Next, you will be asked for your the index of the withdrawal key you wish to use. For **most users**, this will be the default of 0 unless you know what you are doing.
+5. You will then be asked the **validator** **indices** for the validators you wish to generate the message for. You can find your validator indices on block explorers such as [https://beaconcha.in](https://beaconcha.in) or in your Prysm validator client logs. For example, the validator with public key `0x8078c7f4ab6f9eaaf59332b745be8834434af4ab3c741899abcff93563544d2e5a89acf2bec1eda2535610f253f73ee6` on [https://beacocha.in](https://beacocha.in) has validator index 8 by navigating to its [page](https://beaconcha.in/validator/8)
+6. Next you will be asked for your **withdrawal credentials,** which you should now have if you followed this guide
+7. Next you will be asked for the Ethereum address you wish to use to receive your withdrawn funds. This needs to be checksummed, and you can get it from your wallet or a block explorer. **You cannot change this once it is set on-chain**, so triple check it
+
+Below is an example of running through the interactive process explained above:
+
+```
+python ./staking_deposit/deposit.py generate-bls-to-execution-change
+Please choose your language ['1. العربية', '2. ελληνικά', '3. English', '4. Français', '5. Bahasa melayu', '6. Italiano', '7. 日本語', '8. 한국어', '9. Português do Brasil', '10. român', '11. Türkçe', '12. 简体中文']:  [English]: english
+
+Please choose the (mainnet or testnet) network/chain name ['mainnet', 'goerli', 'sepolia', 'zhejiang']:  [mainnet]: zhejiang
+
+Please enter your mnemonic separated by spaces (" "). Note: you only need to enter the first 4 letters of each word if you'd prefer.: 
+bike shoe attitude violin fun life punch enhance attend bright voyage wheel clutch taxi high health siren jealous tell female upon firm manual wage
+
+Please enter the index (key number) of the signing key you want to use with this mnemonic. [0]: 0
```

Comment:
Another question about the staking cli: why this index is singular, while the addresses and indices below are plural. Is this index the "first" index that will be consider and the addresses will be taken sequentially from the lists below, that is, if I put here index 7 and then below I put indices 89, 23, 31. Then the staking cli will produce bls to exec changes for the validators with indices 89, 23 and 31 that were created with this mnemonic with the paths of the form m/12381/3600/7/0/0, m/12381/3600/8/0/0 and m/12381/3600/9/0/0?

Reply:
I'll try to test this...